app/models/job.rb

```diff
@@ -1,2 +1,5 @@
 class Job < ActiveRecord::Base
+  belongs_to :company
+
+  validates :position, :posting_date, :description, :company_id, presence: true
```

Comment:
Do we want to validate the presence of all these? It seems like many postings will have incomplete information, and it might be better to leave them as `null` rather than forcing them to be filled with some arbitrary string like "unknown" or "?".